src/utils/colonyActions.ts

```diff
@@ -323,13 +337,14 @@ export const getSpecificActionValuesCheck = (
     domainColor?: string | null;
     verifiedAddresses?: string[] | null;
     isWhitelistActivated?: boolean | null;
+    colonySafes?: ColonySafe[];
   },
 ): { [key: string]: boolean } => {
   switch (actionType) {
     case ColonyAndExtensionsEvents.ColonyMetadata: {
       const nameChanged = prevColonyDisplayName !== currentColonyDisplayName;
       const logoChanged = prevColonyAvatarHash !== currentColonyAvatarHash;
-
+      const safeAdded = currentColonySafes.length > prevColonySafes.length;
```

Comment:
@chinins For this one, I updated the `colonyAction` resolver so that if `colonySafes` comes back from IPFS as `null` it will return an empty array. This is consistent with how we've typed the colonySafes property (i.e. as non-nullable).